snaps/features/custom-ui/index.md

```diff
@@ -182,6 +182,55 @@ await snap.request({
 });
 ```
 
+### `Banner`
+
+Outputs a banner alert component that can be used to display important messages with different severity levels.
+
+#### Props
+
+- `title`: `string` - Title of the banner.
+- `severity`: `"danger" | "info" | "success" | "warning"` - Severity level of the banner.
+- `children`: The content to display in the banner. Can include:
+  - Text components
+  - Formatting components
+  - Links
+  - Icons
+  - Buttons
+  - Skeleton loaders
```

Comment:
```suggestion
- `title`: `string` - The title of the banner.
- `severity` - The severity level of the banner.
  Possible values are `"danger"`, `"info"`, `"success"`, and `"warning"`.
- `children` - The content to display in the banner.
  This can include text, formatting components, [`Link`](#link) components, [`Icon`](#icon) components, [`Button`](#button) components, and skeleton loaders.
```

Questions: What do you mean by "formatting components"? Is it bold and italic text? Also, are "skeleton loaders" added using the `Image` component?

Reply:
Yes formatting components is bold and italic. Skeleton loaders -- will get back to you about that.

Reply:
Fixed :)